### PR TITLE
Tokenizers should be framework agnostic

### PR DESCRIPTION
**Pull request by LysandreJik**
The `prepare_seq2seq_batch` method should not return PyTorch tensors by default. It does not in the base class, and all our tokenizer methods should be agnostic to the framework.

Updated Marian, Pegasus, mBART, FSMT that had `return_tensors="pt"` and RAG that had `return_tensors="np"`.

The documentation for these methods was inconsistent, added the docstrings via the decorator where it was needed.

### CODE REVIEW

**Comment by LysandreJik** [comment]
Can't ask you to review here @stas00 but would love your review as FSMT is impacted by this.

**Review by sgugger — APPROVED** [review]
Thanks for fixing this!

**Comment by stas00** [comment]
LGTM

File: src/transformers/models/bart/tokenization_bart_fast.py [hunk 1]
```diff
@@ -49,79 +51,18 @@ class BartTokenizerFast(RobertaTokenizerFast):
...
+    @add_start_docstrings(PREPARE_SEQ2SEQ_BATCH_DOCSTRING)
     def prepare_seq2seq_batch(
         self,
         src_texts: List[str],
         tgt_texts: Optional[List[str]] = None,
         max_length: Optional[int] = None,
         max_target_length: Optional[int] = None,
         padding: str = "longest",
-        return_tensors: str = "None",
+        return_tensors: str = None,
```
Comment (patrickvonplaten) [inline src/transformers/models/bart/tokenization_bart_fast.py:62]:
```suggestion
        return_tensors: Optional[str] = None,
```

File: src/transformers/models/fsmt/tokenization_fsmt.py [hunk 2]
```diff
@@ -491,7 +491,7 @@ def prepare_seq2seq_batch(
         tgt_texts: Optional[List[str]] = None,
         max_length: Optional[int] = None,
         max_target_length: Optional[int] = None,
-        return_tensors: str = "pt",
+        return_tensors: str = None,
```
Comment (patrickvonplaten) [inline src/transformers/models/fsmt/tokenization_fsmt.py:494]:
```suggestion
        return_tensors: Optional[str] = None,
```

File: src/transformers/models/marian/tokenization_marian.py [hunk 3]
```diff
@@ -175,7 +175,7 @@ def prepare_seq2seq_batch(
         tgt_texts: Optional[List[str]] = None,
         max_length: Optional[int] = None,
         max_target_length: Optional[int] = None,
-        return_tensors: str = "pt",
+        return_tensors: str = None,
```
Comment (patrickvonplaten) [inline src/transformers/models/marian/tokenization_marian.py:178]:
```suggestion
        return_tensors: Optional[str] = None,
```

File: src/transformers/models/mbart/tokenization_mbart.py [hunk 4]
```diff
@@ -183,7 +183,7 @@ def prepare_seq2seq_batch(
         max_target_length: Optional[int] = None,
         truncation: bool = True,
         padding: str = "longest",
-        return_tensors: str = "pt",
+        return_tensors: str = None,
```
Comment (patrickvonplaten) [inline src/transformers/models/mbart/tokenization_mbart.py:186]:
```suggestion
        return_tensors: Optional[str] = None,
```

**Review by patrickvonplaten — APPROVED** [review]
Awesome, left a tiny nit

**Pull request merged**